aten/src/ATen/native/sparse/cuda/SparseMatMul.cu

```diff
@@ -40,12 +40,30 @@
 #include <thrust/iterator/discard_iterator.h>
 
 
-#if defined(__CUDACC__) && (CUSPARSE_VERSION >= 11000)
+#if defined(__CUDACC__) && ((CUSPARSE_VERSION >= 11000) || (defined(USE_ROCM) && ROCM_VERSION >= 60300))
 #define IS_CUSPARSE11_AVAILABLE() 1
 #else
 #define IS_CUSPARSE11_AVAILABLE() 0
 #endif
 
+#if (IS_CUSPARSE11_AVAILABLE() && defined(USE_ROCM) && (ROCM_VERSION >= 70000))
+#define HIPSPARSE_FP16_SUPPORT 1
+#else
+#define HIPSPARSE_FP16_SUPPORT 0
+#endif
+
+#if (IS_CUSPARSE11_AVAILABLE() && defined(USE_ROCM) && (ROCM_VERSION >= 70000))
+#define HIPSPARSE_FP16_BF16_SUPPORT 1
+#else
+#define HIPSPARSE_FP16_BF16_SUPPORT 0
+#endif
+
+#if (IS_CUSPARSE11_AVAILABLE() && defined(USE_ROCM) && (ROCM_VERSION >= 60300))
```

Comment:
The `IS_CUSPARSE11_AVAILABLE() &&` part is redundant since the later condition subsumes the former anyway